packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/index.js

```diff
@@ -74,7 +83,23 @@ function mapDispatchToProps(dispatch) {
     const action = actions.setSpanNameColumnWidth(...args);
     return dispatch(action);
   };
-  return { setSpanNameColumnWidth };
+  const expandAll = () => {
+    const action = actions.expandAll();
+    return dispatch(action);
+  };
+  const expandOne = spans => {
+    const action = actions.expandOne(spans);
+    return dispatch(action);
+  };
+  const collapseAll = spans => {
+    const action = actions.collapseAll(spans);
+    return dispatch(action);
+  };
+  const collapseOne = spans => {
+    const action = actions.collapseOne(spans);
```

Comment:
Typically, `bindActionCreators` from `redux` would be used to do this. Not sure why it's not used on line 83. Please change to use `bindActionCreators`.